src/NLogShipper/LogzioTarget.cs

```diff
@@ -30,6 +32,8 @@ public class LogzioTarget : TargetWithContext
         public bool UseGzip { get { return _shipperOptions.BulkSenderOptions.UseGzip; } set { _shipperOptions.BulkSenderOptions.UseGzip = value; } }
         public string ProxyAddress { get { return _shipperOptions.BulkSenderOptions.ProxyAddress; } set { _shipperOptions.BulkSenderOptions.ProxyAddress = value; } }
 
+        public string Format { get { return _shipperOptions.BulkSenderOptions.Format; } set { _shipperOptions.BulkSenderOptions.Format = value; } }
```

Comment:
Instead of calling it `Format` and using magic-strings like `text` and `json`. Then just call it `EnableJsonLayout`, see also: https://github.com/googleapis/google-cloud-dotnet/pull/5635

Reply:
I parse specifically the message field here, not the entire payload.
Since json layout works a bit different (appends to given field and not straight to the log) i don't want to create confusion with the name.
I can change the format to boolean if you prefer.

Reply:
Maybe change to a boolean called `EnableJsonMessage` (Skip constant `ToLower` and string-comparisons)